packages/client/src/components/SafeOwners.js

```diff
@@ -13,9 +13,10 @@ function SafeOwners({
     setSafeOwners([...newOwners]);
   };
 
-  const onOwnerAddressChange = (value, idx) => {
+  const onOwnerAddressChange = (value, isValid, idx) => {
     const newOwners = safeOwners.slice(0);
     newOwners[idx].address = value;
+    newOwners[idx].isValid = isValid;
     setSafeOwners([...newOwners]);
   };
```

Comment:
Prefer following the style guide
```suggestion
  const onOwnerAddressChange = (value, isValid, idx) => {
    const newOwners = [...safeOwners]
    newOwners[idx].address = value;
    newOwners[idx].isValid = isValid;
    setSafeOwners(newOwners);
  };
```